### ISSUE TITLE
RSX: FS build failed: error X3018: invalid subscript 'ssa' - [NPEB00646]

### ISSUE THREAD

**Issue by MyaniPT** [RPCS3/rpcs3#1488]
```
{rsx::thread} RSX: Unknown/illegal instruction: 0x17 (forced unit 0)
{rsx::thread} RSX: Unknown/illegal instruction: 0x17 (forced unit 0)
{rsx::thread} RSX: Unknown/illegal instruction: 0x17 (forced unit 0)
{rsx::thread} RSX: Unknown/illegal instruction: 0x17 (forced unit 0)
{PPU Thread[0x65] (veThread3)[0x00647420]} sys_fs: sys_fs_stat('/dev_hdd0/game/NPEB00646/USRDIR/Sounds/eng/74F576D7D49BC601.str') failed: not found
{rsx::thread} RSX: FS build failed:C:\Users\Annie\Desktop\rpcs3\FragmentProgram.hlsl(79,15-20): error X3018: invalid subscript 'ssa'
```

DirectX 12
PPU Interpreter 2
SPU Interpreter 2

**LLE:**
cellFiber
cellSre
cellSpursJq

Log and .hlsl files @ [rpcs3.zip](https://github.com/RPCS3/rpcs3/files/121486/rpcs3.zip)

BurgerTime World Tour **[NPEB00646]**

Game boots and shows three different screens, doesn't get to menu, crashes with unknown exception.

**Comment by tambry** [comment]
Probably due to unknown shader instructions.

**Comment by MyaniPT** [comment]
```
·E {rsx::thread} RSX: Src type 3 used, please report this to a developer.
·E {rsx::thread} RSX: Src type 3 used, please report this to a developer.
·E {rsx::thread} RSX: Src type 3 used, please report this to a developer.
·E {rsx::thread} RSX: Src type 3 used, please report this to a developer.
·E {rsx::thread} RSX: Src type 3 used, please report this to a developer.
·E {rsx::thread} RSX: Src type 3 used, please report this to a developer.
·E {rsx::thread} RSX: FS build failed:C:\Users\Annie\Desktop\rpcs3_482\FragmentProgram.hlsl(81,28): error X3000: syntax error: unexpected token '.'
C:\Users\Annie\Desktop\rpcs3_482\FragmentProgram.hlsl(81,24-33): error X3013: 'abs': no matching 0 parameter intrinsic function
C:\Users\Annie\Desktop\rpcs3_482\FragmentProgram.hlsl(81,24-33): error X3013: Possible intrinsic functions are:
C:\Users\Annie\Desktop\rpcs3_482\FragmentProgram.hlsl(81,24-33): error X3013:     abs(float|half|double|min10float|min16float|int|uint|min12int|min16int|min16uint)
```

**NPEA00206** - Swords and Soldiers
Similar error, the "please report this to a developer" inside the RSX warning called my attention.
I can't upload the zipped log for some reason (???) github not letting me

First time I tested this happened: http://i.imgur.com/PM3HHYc.png
I can't make it happen again though, it crashes the game before starting, in first time it booted to the menu I believe.

**Comment by vlj** [comment]
I'm on the issue, it's fixed in my rsx branch but I'm having some issue displaying the screen menu.
The scene is correctly draw but for some reason one of the postprocessing effect only draw a black quads.

**Comment by MyaniPT** [comment]
Gotcha.
Should I comment here whenever I get RSX different error codes (other than X3000, X3013 and X3018)?

**Comment by MyaniPT** [comment]
This has been fixed with #1519 
[rpcs3_1519.zip](https://github.com/RPCS3/rpcs3/files/149334/rpcs3_1519.zip)

DX12
XAudio2
PPU 2 / SPU 2
cellSre

![screenshot_79](https://cloud.githubusercontent.com/assets/10283761/13373070/da7ef092-dd54-11e5-8ee2-4445784c93c5.png)

Menu doesn't respond to X or O commands. I can use the arrows to select stuff but if I press X it does nothing, nor does O (K and L respectively on the keyboard).

If nothing else, this can be closed.

**Comment by MyaniPT** [comment]
This has been fixed, closing
